site/src/pages/TemplatePage/TemplateInsightsPage/TemplateInsightsPage.tsx

```diff
@@ -261,6 +281,209 @@ const TemplateUsagePanel = ({
   )
 }
 
+const TemplateParametersUsagePanel = ({
+  data,
+  ...panelProps
+}: PanelProps & {
+  data: TemplateInsightsResponse["report"]["parameters_usage"] | undefined
+}) => {
+  return (
+    <Panel {...panelProps}>
+      <PanelHeader>
+        <PanelTitle>Parameters usage</PanelTitle>
+        <PanelSubtitle>Last 7 days</PanelSubtitle>
+      </PanelHeader>
+      <PanelContent>
+        {!data && <Loader sx={{ height: 200 }} />}
+        {data && data.length === 0 && <NoDataAvailable sx={{ height: 200 }} />}
+        {data &&
+          data.length > 0 &&
+          data.map((parameter, parameterIndex) => {
+            const label =
+              parameter.display_name !== ""
+                ? parameter.display_name
+                : parameter.name
+            return (
+              <Box
+                key={parameter.name}
+                sx={{
+                  display: "flex",
+                  alignItems: "start",
+                  p: 3,
+                  marginX: -3,
+                  borderTop: (theme) => `1px solid ${theme.palette.divider}`,
+                  width: (theme) => `calc(100% + ${theme.spacing(6)})`,
+                  "&:first-child": {
+                    borderTop: 0,
+                  },
+                }}
+              >
+                <Box sx={{ fontWeight: 500, flex: 1 }}>{label}</Box>
+                <Box sx={{ flex: 1, fontSize: 14 }}>
+                  {parameter.values
+                    .sort((a, b) => b.count - a.count)
+                    .map((value, valueIndex) => (
+                      <Box
+                        key={`${parameterIndex}-${valueIndex}`}
+                        sx={{
+                          display: "flex",
+                          alignItems: "baseline",
+                          justifyContent: "space-between",
+                          py: 0.5,
+                          gap: 5,
+                        }}
+                      >
+                        <ValueLabel value={value} parameter={parameter} />
```

Comment:
This could be then `<ParameterUsageValue type={string} value={string} count={int64}>`

Reply:
Why should we use 3 props instead of 2? I just use what is easier for implementation which usually makes things easier.

Reply:
I assumed that `ParameterUsageValue` represents also the counter, but if it is a separate column, then we can leave it.

The main problem I see with `ValueLabel` is confusing var names like `value.value.trim()`. I'd rather see this as `parameterUsage.value. ...`.

Reply:
We can definitely change the name for `parameterUsage`, I also found it better to read.